Toggl.Droid/Autocomplete/AutocompleteExtensions.cs

```diff
@@ -19,6 +19,17 @@ public static class AutocompleteExtensions
         public static IImmutableList<ISpan> AsImmutableSpans(this ICharSequence text, int cursorPosition)
             => text.AsSpans(cursorPosition).ToImmutableList();
 
+        public static bool isEqualTo(this ISpannable first, ISpannable second)
```

Comment:
Don't you `camelCase` me! 😋 

```suggestion
        public static bool IsEqualTo(this ISpannable first, ISpannable second)
```